### ISSUE TITLE
Add title to schema

### ISSUE THREAD

**Issue by zklaus** [clix-meta/clix-meta#25]
In #20, @aulemahal suggested to add a title to the index definition:

> Indicator.title A short human readable string that describes the indicators, without “placeholders” (like no "{TT}")

I have two questions.

1. With regards to the placeholders ("{TT}"), the idea is that this is a kind of meta index, where a specific index can be "instantiated" by replacing the placeholders everywhere. In this way, it makes a lot of sense to me to keep the placeholders also in something like "title", since that means that the information ending up in the final data file describes everything in a meaningful way. @aulemahal, do you still feel it is important to have the title without placeholders?

2. Looking at the table, it seems that `OUTPUT_long_name` fulfills a similar purpose. What would be the difference between title and `OUTPUT_long_name`?

**Comment by aulemahal** [comment]
Well, my idea was that the title would be almost the same as the long_name, but completely human-readable. By that I mean, for example, for `tmlt{PRC}p`: 
 - long_name : `Percentage of days when Tmean < {PRC}th percentile`
 - title : `Percentage of days when the mean daily temperature is under a given percentile`.

But if you feel this is too much repetition (especially for indices with no "placeholders"), I don't care thjat much about this.

**Comment by zklaus** [comment]
In principle, the long_name should already be human readable, but there may be two issues. One is a possible lack of clarity, so I think it would be nice to address these.

The other is the question of whether things should be understandable before or after replacing the placeholders with their actual values. We have worked under the assumption that everything is clear only after replacing the placeholders.

Overall, I think that for indices without placeholders `long_name` should serve the function you suggest without a problem. For indices with placeholders, perhaps the question is rather how we want to use them and what purpose the placeholders serve.

**Comment by larsbarring** [comment]
Is there CMIP6 recommendations for long names that we could draw on? I think that I have seen a draft at some point. But this will of course not address the issue of replacing `{..}` with something more general and easy to understand.

**Comment by zklaus** [comment]
Well, so-called MIP variables have a title attribute that should be used for the `long_name`. For a list of MIP variables see [here](http://clipc-services.ceda.ac.uk/dreq/index/var.html).

**Comment by larsbarring** [comment]
Yes, but I was aiming at was not the long names themselves, but rather a style guide for how to construct them.

**Comment by larsbarring** [comment]
@aulemahal your request was for a 'title' that may guide users of software using clix-meta rather than being something to put into the file produced?

**Comment by aulemahal** [comment]
Yes exactly! I am thinking of some short description that gives a clear idea to the user about what the indice does. But it is not included in the output as the formatted long_name should already playl this role.

**Comment by martinjuckes** [comment]
In answer to Lars's question [above](https://github.com/clix-meta/clix-meta/issues/25#issuecomment-828391685) about CMIP6 long names, there is a [Style Guide for Variable Titles in CMIP6](https://zenodo.org/record/2480853).

We didn't have the concept of placeholders in CMIP6 long names. For example, we have variables with descriptions "Temperature on the 500 hPa surface" and "Air temperature at 700hPa" both have long name "Air Temperature". Perhaps we should use placeholders within the database to improve clarity.
